test/metabase/db/custom_migrations_test.clj

```diff
@@ -2286,33 +2287,36 @@
             multi-stage-dashcard2-id (create-dashcard multi-stage-question-id [])
             multi-stage-dashcard3-id (create-dashcard multi-stage-question-id)
             multi-stage-model-dashcard-id (create-dashcard multi-stage-model-id)
-            stage-0-pattern [{:card_id 1,
-                              :parameter_id string?
-                              :target ["dimension" ["field" 200 {:base-type "type/Integer"}] {:stage-number 0}]}
-                             {:card_id 1
-                              :parameter_id string?
-                              :target ["strange-long-forgotten-target" ["field" "count" {:base-type "type/Integer"}]]}
-                             {:card_id 1
-                              :parameter_id string?
-                              :target ["dimension" ["field" 275 {:base-type "type/Text"}] {:stage-number 0}]}]
-            stage-2-pattern [{:card_id 1
-                              :parameter_id string?
-                              :target ["dimension" ["field" 200 {:base-type "type/Integer"}] {:stage-number 2}]}
-                             {:card_id 1
-                              :parameter_id string?
-                              :target ["strange-long-forgotten-target" ["field" "count" {:base-type "type/Integer"}]]}
-                             {:card_id 1
-                              :parameter_id string?
-                              :target ["dimension" ["field" 275 {:base-type "type/Text"}] {:stage-number 2}]}]
-            no-stage-pattern [{:card_id 1
-                               :parameter_id string?
-                               :target ["dimension" ["field" 200 {:base-type "type/Integer"}]]}
-                              {:card_id 1
-                               :parameter_id string?
-                               :target ["strange-long-forgotten-target" ["field" "count" {:base-type "type/Integer"}]]}
-                              {:card_id 1
-                               :parameter_id string?
-                               :target ["dimension" ["field" 275 {:base-type "type/Text"}]]}]
+            stage-0-pattern (fn [card-id]
```

Comment:
Since we're not concerned much with this value, I think it's fine to just use `:int?` since we're using it with `=?`

Reply:
I prefer to make it obvious that those `int`s are the same.